jest/babel.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = api => {
```

Comment:
can we use the existing babel config or perhaps extend it? In Panther we do something similar and I'd rather have a single babel configuration